CAN_haz_bus/CAN_haz_bus.ino

```diff
@@ -168,8 +169,20 @@ KEEP_TRYING:
                         (inbuf[2] << 24);
                 extflag = inbuf[6];
                 len = inbuf[0] - 7;
-                results = CAN.sendMsgBuf(canId, extflag, len, inbuf+7);
+                failCnt = 0;
```

Comment:
i like the retrans up to 10 times.  in the future, we should make 10 a settable number.